src/api/ethereum.ts

```diff
@@ -130,33 +129,19 @@ export const startFlow = async (
   });
   let call = [];
 
-  const isSubscribedUSDC = await idaContract.methods
-    .getSubscription(
-      USDCxAddress,
-      twoWayMarketAddress, // publisher
```

Comment:
This variable is imported, use`exchangeAddress` which in this context will be the `twoWayMarketAddress`